frontend/src/common/get24HourTime.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ *
+ * @param eventStartTime date time number
```

Comment:
consider calling this param something more generic like dateTime or unixDate etc

Reply:
Resolved in commit c250da81f5f68f938da6eb63dc522128055245a8